Zina1/Bash/proces.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+function getcpid {
+  tp=`ps x -o %c $1`        #get childs pids of parent pid
+  for i in $tp; do          #loop through childs
+    if [ -z $i ]; then      #check if empty list
+      exit                  #if empty: exit
+    else                    #else
+      echo -n "$i "         #print childs pid
+    fi;
+  done
+}
+
+getcpid $1
```

Comment:
Getting the following error 

`error: unknown user-defined format specifier "%c"`